src/Bicep.MSBuild.E2eTests/src/simpleMultiTargeting.test.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { Example } from "./example";
+
+describe("msbuild", () => {
+  it("should build a multi-targeting project with default output paths successfully", () => {
+    const example = new Example("simpleMultiTarget");
+    example.cleanProjectDir();
+
+    const buildResult = example.build();
+    expect(buildResult.stderr).toBe("");
+
+    example.expectTemplate("bin/Debug/net7.0/empty.json");
+    example.expectTemplate("bin/Debug/net7.0/passthrough.json");
+    example.expectTemplate("bin/Debug/net7.0/theAnswer.json");
```

Comment:
Can users alias included templates? Or alternatively, could we replicate the template directory structure?

Not sure if this is germane to this PR, but the behavior when compiling something like the following should be controllable:

./main.bicep
./subdir/main.bicep

Reply:
The behavior we shipped with was to default to the output directory without preserving the directory structure unfortunately. There is ability to customize it via the `OutputFile` metadata, but the user has to specify that explicitly.